app/assets/javascripts/event-stage.js

```diff
@@ -376,7 +379,7 @@
             post: function() {
                 var self = this;
                 var creating = this.__new__;
-                return this.request( 'staging/' + this.slug + '/post_event_course_org', 'POST', 'application/json' )
+                return this.request( 'staging/' + (creating ? 'new' : this.slug) + '/post_event_course_org', 'POST', 'application/json' )
```

Comment:
I would prefer to post using the id instead of the slug.

Reply:
Ah. I did mean to do that one. Thanks.